lib/versioning/npm/range.ts

```diff
@@ -17,6 +17,12 @@ export function getNewValue(
   fromVersion: string,
   toVersion: string
 ): string {
+  if (rangeStrategy === 'cap') {
+    if (isVersion(currentValue)) {
+      return `<= ${currentValue}`;
```

Comment:
If it's already a version then it's capped and we don't need to do anything - we should return `toVersion` like `pin` does. Our goal is not to have everything as ranges - it's just to convert from uncapped ranges to capped ranges.